ASM/c/main.c

```diff
@@ -48,19 +47,30 @@ void before_game_state_update() {
     display_misc_messages();
 }
 
+void close_rando_display_buffer() {
+    OPEN_DISPS(z64_ctxt.gfx);
+
```

Comment:
recommend adding something to force a crash if rando_db->p exceeds the size. something like

```suggestion
if(((int)rando_db->p - (int)rando_db->buf) > rando_db->size) {
        *((int*)0) = 0;
    }
```

Reply:
Even though it won't display on VC, I've got a better idea... (should say rando display buffer though, copy and paste error,  same with the "size" value)

![Screenshot 2024-07-15 21-09-10](https://github.com/user-attachments/assets/d25f80f6-9caa-4f01-bf38-04027a6db221)